tests/app/main/test_frameworks.py

```diff
@@ -3395,94 +3392,53 @@ def _send_email(self, clarification_question):
+                                 "clarification_question": clarification_question}
             )
 
         if succeeds:
             notify_send_email.assert_any_call(
                 mock.ANY,  # DMNotifyClient
-                "email@email.com",
-                template_name_or_id=mock.ANY,
+                to_email_address="email@email.com",
+                template_name_or_id='1a8a3408-49ef-486f-a6c9-8557d1a0dc63',
                 personalisation={'user_name': 'Năme', 'framework_name': 'Test Framework',
                                  'clarification_question_text': clarification_question},
-                reference=mock.ANY,
-                reply_to_address_id=mock.ANY
+                reference='clarification-question-confirm-8yc90Y2VvBnVHT5jVuSmeebxOCRJcnKicOe7VAsKu50=',
+                reply_to_address_id='24908180-b64e-513d-ab48-fdca677cec52'
             )
 
-    def _assert_application_email(self, mandrill_send_email, *email_body_content, succeeds=True):
-
-        if succeeds:
-            assert mandrill_send_email.call_count == 1
-        else:
-            assert mandrill_send_email.call_count == 0
-
-        if succeeds:
-            mandrill_send_email.assert_called_with(
-                mock.ANY,  # self
-                to_email_addresses="digitalmarketplace@mailinator.com",
-                from_email_address="enquiries@digitalmarketplace.service.gov.uk",
-                from_name="Test Framework Supplier",
-                subject="Test Framework application question",
-                email_body=FakeMail(*email_body_content),
-                tags=["application-question"],
-                reply_to="email@email.com",
-            )
-
-    def test_should_call_send_email_with_correct_params(self, mandrill_send_email, notify_send_email, s3):
+    @mock.patch('dmutils.s3.S3')
+    def test_should_call_send_email_with_correct_params(self, s3, notify_send_email):
         self.data_api_client.get_framework.return_value = self.framework('open', name='Test Framework')
 
         clarification_question = 'This is a clarification question.'
         response = self._send_email(clarification_question)
 
-        self._assert_clarification_email(mandrill_send_email, notify_send_email,
-                                         clarification_question=clarification_question)
+        self._assert_clarification_email(notify_send_email,
+                                         clarification_question=clarification_question, times_called=2)
 
         assert response.status_code == 200
         assert self.strip_all_whitespace(
             '<p class="banner-message">Your clarification question has been sent. Answers to all ' +
             'clarification questions will be published on this page.</p>'
         ) in self.strip_all_whitespace(response.get_data(as_text=True))
 
-    def test_should_call_send_g7_email_with_correct_params(self, mandrill_send_email, notify_send_email, s3):
```

Comment:
Is this test G7 specific? I think it's the only place we cover the scenario where `clarification_questions_open=False` and we send the framework-application email. Can we replace it with G10 and see what happens?